### PR TITLE
Use CDN to load TinyMCE [stage-2]

### PR DESCRIPTION
**Pull request by stulees**
- The minify process of our widget was causing an issue on RVA when including the tinymce source as part of the concatenated source build. Now using CDN to load instead
- Removing gulp task regarding tinymce which no longer adds tinymce assets to dist/ as they are no longer needed

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review. This has been deployed to stage-2 but I uploaded a production version to my own test bucket. You can validate that the settings open and work normally with this presentation:

http://rva.risevision.com/#/PRESENTATION_MANAGE/id=0bb25dc1-a902-4fa6-90de-1642387cb94b?cid=30007b45-3df0-4c7b-9f7f-7d8ce6443013

**Comment by donnapep** [comment]
@stulees I think perhaps we should add a fallback to a local version so this will work offline once we get to that point.

**Comment by stulees** [comment]
@donnapep A fallback will mean we need to include all the tinymce assets in the dist/ folder (plugins, skins, themes). Do you want to continue keeping them in dist/?

**Comment by donnapep** [comment]
@stulees Yes, I think we should copy the dependencies over from the `widget-settings-ui-components` repo and put them into `dist`. Do you see a reason not to?

**Comment by stulees** [comment]
@donnapep Yes I found that to be unnecessary as all `widget-settings-ui-components` bower dependencies already get brought in and placed within `src/components/ when the widget bower file has `widget-settings-ui-components` as a dependency. This is the case within any dependency, it will always pull in whatever is in its bower file. 

The dependencies that I'm speaking to are not source files but tinymce assets which get loaded at runtime by tinymce and are required locally if we're not using CDN. I'll provide a fallback and add the gulp task back in for adding all tinymce assets.

**Comment by stulees** [comment]
@donnapep I remember borrowing this block of code from Image Widget - https://github.com/Rise-Vision/widget-time-date/blob/master/src/settings.html#L37-L39 . Do you recall if this worked? The fallback points to `components/..` but `components/..` won't exist in the `dist/..` folder ...

**Comment by donnapep** [comment]
@stulees Image has a `components` folder but no `angular` folder. We should try to remember to fix that next time we're in the Image or Date and Time Widget.

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review last commit. I've revised and added fallback path locations for Angular, TinyMCE, and jQuery and added a gulp task "vendor" which will put these dependencies to a `dist/js/vendor` folder so we have them as a fallback to CDNs failing. 

You can also still validate with the previous presentation I provided (a production version of the widget in my test bucket)

File: src/settings.html [hunk 1]
```diff
@@ -36,8 +36,14 @@
 
   <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.3.20/angular.min.js"></script>
   <!-- if AngularJS fails to load fallback to a local version -->
-  <script>window.angular || document.write(unescape("%3Cscript src='components/angular/angular.min.js' type='text/javascript'%3E%3C/script%3E"));
+  <script>window.angular || document.write(unescape("%3Cscript src='js/vendor/angular/angular.min.js' type='text/javascript'%3E%3C/script%3E"));
   </script>
+
+  <script src="//cdn.tinymce.com/4/tinymce.min.js"></script>
+  <!-- if TinyMCE fails to load fallback to a local version -->
+  <script>window.angular || document.write(unescape("%3Cscript src='js/vendor/tinymce-dist/tinymce.min.js' type='text/javascript'%3E%3C/script%3E"));
```
Comment (donnapep) [inline src/settings.html]: @stulees I believe this should be checking for the TinyMCE object here and not `window.angular`. Thx.

**Comment by stulees** [comment]
@donnapep Ack, copy/paste mishap, good catch. Have now corrected.

**Comment by donnapep** [comment]
@stulees Yup, happens to everyone. :+1:

**Pull request merged**